models/players.js

```diff
@@ -15,13 +15,24 @@ var parseString = require('xml2js').parseString;
 var players = [];
 var encryption = require('../encryption.js');
 var shortId = require('shortid');
+var team_colors_nba = require('../lists/team_colors/team_colors_nba.js')
 
 
-var returnPlayers = function (players, rb_team_id, res, league){
+var intreturnPlayers = function(players, rb_team_id, res, league, second_callback){
```

Comment:
what does this function name mean? Why isnt it camelcase? Also, why is it needed? It doesnt really do anything,